qa/tasks/cephfs/test_scrub.py

```diff
@@ -144,6 +147,34 @@ def _scrub(self, workload, workers=1):
                 errors[0].exception, errors[0].backtrace
             ))
 
+    def _get_damage_count(self, damage_type='backtrace'):
+        out_json = self.fs.rank_tell(["damage", "ls"])
+        self.assertNotEqual(out_json, None)
+
+        damage_count = 0
+        for it in out_json:
+            if it['damage_type'] == damage_type:
+                damage_count += 1
+        return damage_count
+
+    def _scrub_new_files(self, workload):
+        """
+        That scrubbing new files does not lead to errors
+        """
+        workload.create_files(1000)
+
+        self._wait_until_scrub_complete()
+
+        damage_count = self._get_damage_count()
```

Comment:
nit: better would be:

`self.assertEqual(self._get_damage_count(), 0)`

Reply:
> nit: better would be:
> 
> `self.assertEqual(self._get_damage_count(), 0)`

ack ... fixed it